sim/mage/talents_arcane.go

```diff
@@ -311,6 +311,9 @@ func (mage *Mage) registerArcanePowerCD() {
 			}
 			mage.arcanePowerCostMod.Deactivate()
 			arcanePowerDmgMod.Deactivate()
+			if mage.t13ProcAura != nil {
+				mage.t13ProcAura.SetStacks(sim, 0)
```

Comment:
You should call `Deactivate()` here (which will automatically set the stacks to 0) rather than only `SetStacks()`.

Reply:
Would that not also remove the buff from the timeline until your next cast? The way I understood it, it will reset the buff to 0 stacks.
I guess it doesnt really matter for it to work and .Deactivate looks cleaner